src/main/java/com/worldwarofants/game/util/RandGen.java

```diff
@@ -0,0 +1,77 @@
+package com.worldwarofants.game.util;
+
+import java.util.Random;
+
+/*
+    Utility class that provides an abstraction of Java's Random class.
+    Its goal is to provide developers with a simple way to work with
+    random number generation, as well as allow for precise tweaking
+    of exact methods used.
+
+    Please make sure to generate your random numbers within the project
+    exclusively using this package.
+*/
+
+public class RandGen extends java.util.Random {
+    private static final long serialVersionUID = -5378246343912387722L;
+
+    Random generator;
+    long seed;
+    String seedString; // If a custom text input has been provided by user and hashed,
+
+    // Overloaded seedless constructor
+    public RandGen() {
```

Comment:
These are pointless comments that don't explain anything that the code doesn't say by itself. This should be avoided because when you have to change the code, you also have to change the comments too, so now you have to maintain another extra thing.

Comments should be used to explain how to use your classes and methods, and WHY you chose to implement it in that specific way, not what the code does. If you think you need comments on what the code does, then that code isn't written well. If the code isn't written well because you didn't have a choice, then you can add comments.

Reply:
They've been replaced with hopefully better javadocs.